api_reference.md

```diff
@@ -607,19 +622,26 @@ param = {
   }
 }
 
-charge_response = Midtrans.update_subscription("d137e7f4-9474-4fc2-9847-672e09cb16f6", param)
+response = Midtrans.update_subscription("d137e7f4-9474-4fc2-9847-672e09cb16f6", param)
 
-charge_response.body == {
+# this will be Hash representation of the API JSON response:
+puts response.data == {
         "status_message": "Subscription is updated."
 }
 ```
 
-#### `Midtrans::Result`
-
+## Get json field value from Midtrans API result
 ```ruby
 result = Midtrans.charge(...)
 
-result.class # => Midtrans::Result
+# If in the future there is a new attribute in the Midtrans Result.
+# Recommended retrieve like example below.
+# :new_attribute is the new field name in API response JSON
+result.data[:new_attribute]
+
+# Alternatively you can also access like below.
+# But make sure the new field name already show in API response JSON, if not will trigger error.
```

Comment:
Grammar
> But please make sure the new field name is actually exists on the API response JSON, else it will trigger an error.

or better wording
> Note: When the field name is not actually exists on the API response JSON, it will trigger an error. Make sure to handle it.

Reply:
Which remind me of: you don't even document handling error / exception. Didn't I tell you to refer to the NodeJS repo, which it have that section right? Why not adding it on this repo? At least demonstrate what kind of attributes is accessible from the error object